docs/DOCKER.rst

```diff
@@ -47,19 +39,28 @@ For running an S3 AWS backend, you will have to add a new section
             "credentialsProfile": "default"
         }
     }
+(...)
 
-You will also have to mount your AWS credentials file:
+You will also have to edit your AWS credentials file to be able to use your
+command line tool of choice. This file should only state credentials for S3 AWS
+hosted data, not for your Zenko CloudServer hosted data.
+As for your locationConfig.json, you will need to mount your AWS credentials
```

Comment:
Just as you have to mount your locationConfig.json, you will also need to mount your aws credentials file.

Reply:
Fixed